SwiftLeeds/Data/Model/Sponsor.swift

```diff
@@ -18,10 +18,19 @@ struct Sponsor: Decodable, Hashable, Identifiable {
     let sponsorLevel: SponsorLevel
     let url: String
     let id: String
+    let jobs: [Job]
 }
 
 enum SponsorLevel: String, Decodable {
     case silver
     case platinum
     case gold
 }
+
+struct Job: Decodable, Hashable, Identifiable {
+    let details: String
+    let id: String
```

Comment:
This should be a UUID not String

Reply:
Good catch 😀
Addressed [String to UUID](https://github.com/SwiftLeeds/swiftleeds-ios/pull/44/commits/2e1eba0bd071555dddb3df94fa80d41501fd4431)